src/components/DocumentStatus/DocumentStatus.tsx

```diff
@@ -79,7 +77,7 @@ const DocumentStatusStyles = styled.div`
   }
 
   .issuedby {
-    ${tw`text-gray-700 text-lg font-semibold`}
+    ${tw`text-cloud-900 text-lg font-semibold`}
 
```

Comment:
can shift

Reply:
done, shifted tw to div in some of the other files too.